app/services/importer/get_relevant_job_urls.rb

```diff
@@ -19,14 +23,20 @@ def fetch_jobs
       jobs
     end
 
+    private
+
     def return_relevant_jobs
       data = @ats.fetch_company_jobs(@ats_identifier)
       relevant_jobs = []
-      data&.each do |job|
-        title, location = @ats.fetch_title_and_location(job)
-        relevant_jobs << @ats.fetch_url(job) if relevant?(title, location)
+      data&.each do |job_data|
+        title, location = @ats.fetch_title_and_location(job_data)
+        relevant_jobs << @ats.fetch_posting_url(job_data, *additional_args) if relevant?(title, location)
       end
       relevant_jobs
     end
+
+    def additional_args
```

Comment:
What's the story here?

Reply:
Okay, I made the name more explanatory and added a comment.